### PR TITLE
Add assertions to Studio guide

### PR DESCRIPTION
**Pull request by panzarino**
Updates studio guide to include assertions in the "extend test" portion - updates all screenshots in that guide for consistency

### CODE REVIEW

**Comment by jennifer-shehane** [comment]
Changing the base branch to be against 8.1.0 release branch.

File: content/guides/core-concepts/cypress-studio.md [hunk 1]
```diff
@@ -229,12 +276,17 @@ describe('Cypress Studio Demo', () => {
...
+[cypress/tests/demo/cypress-studio.spec.ts](https://github.com/cypress-io/cypress-realworld-app/blob/develop/cypress/tests/demo/cypress-studio.spec.ts)
+file.
 
 </Alert>
 
 ## History
 
 | Version                                     | Changes                              |
 | ------------------------------------------- | ------------------------------------ |
+| [8.1.0](/guides/references/changelog#8-1-0) | Added assertions to guide            |
```
Comment (jennifer-shehane) [inline content/guides/core-concepts/cypress-studio.md:291]: This is a history of the product changes for the feature, not changes to the docs themselves.

```suggestion
| [8.1.0](/guides/references/changelog#8-1-0) | Added ability to generate assertions            |
```

**Review by jennifer-shehane — APPROVED** [review]
Content looks good to me.

File: content/guides/core-concepts/cypress-studio.md (hunk 1 above)
Comment (panzarino) [reply content/guides/core-concepts/cypress-studio.md:291]: Ah thanks, didn't know that

**Pull request merged**